Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "phoenix-core"
-version = "0.11.0-rc.0"
+version = "0.11.0-rc.1"
```

Comment:
Bump on a sepparated commit once this gets merged. This is the procedure we always use.